packages/dialtone-vue2/components/stack/stack.vue

```diff
@@ -1,10 +1,13 @@
 <template>
   <component
     :is="as"
+    data-qa="dt-stack"
```

Comment:
Added this `data-qa` attribute, even though from what I can tell it's not used at all. Just following convention across all (or at least most) components and recipes.